apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -3,72 +3,130 @@
+          {% for subject in course.subjects.all %}
+            {# If the current page is a draft, show draft subjects with a class annotation for styling #}
+            {% if current_page.publisher_is_draft %}
+              {% if subject.public_extension %}
+                <li class="course-detail__content__subjects__item">
+                  {{ subject.public_extension.extended_object.get_title }}
+                </li>
+              {% else %}
+                <li class="course-detail__content__subjects__item--draft">
+                  {{ subject.extended_object.get_title }}
+                </li>
+              {% endif %}
+            {# If the current course page is the published version, show only the subjects that are published #}
+            {% elif subject.public_extension %}
+              <li class="course-detail__content__subjects__item">
+                {{ subject.public_extension.extended_object.get_title }}
+              </li>
+            {% endif %}
+          {% empty %}
+            <li class="course-detail__content__subjects--empty">
+              {% trans "No associated subjects" %}
             </li>
-          {% endif %}
-        {# If the current page is the published version, show only the organizations that are published #}
-        {% elif organization.public_extension %}
-          <li class="course-detail__organizations__item">
-            {{ organization.public_extension.extended_object.get_title }}
-          </li>
-        {% endif %}
-      {% endfor %}
-    </ul>
-
-    <div class="course-detail__active-session">
+          {% endfor %}
+        </ul>
+      </div>
+
+      <div class="course-detail__content__organizations">
+        <h2>{% trans 'Organizations' %}</h2>
+        <ul class="course-detail__content__organizations__list">
+          {% for organization in course.organizations.all %}
+            {# If the current page is a draft, show draft organizations with a class annotation for styling #}
+            {% if current_page.publisher_is_draft %}
+              {% if organization.public_extension %}
+                <li class="course-detail__content__organizations__item">
+                  {{ organization.public_extension.extended_object.get_title }}
+                </li>
+              {% else %}
+                <li class="course-detail__content__organizations__item--draft">
+                  {{ organization.extended_object.get_title }}
+                </li>
+              {% endif %}
+            {# If the current page is the published version, show only the organizations that are published #}
+            {% elif organization.public_extension %}
+              <li class="course-detail__content__organizations__item">
+                {{ organization.public_extension.extended_object.get_title }}
+              </li>
+            {% endif %}
+          {% endfor %}
+        </ul>
+      </div>
+
+    <div class="course-detail__content__information">
```

Comment:
is there a way to wrap this diff in "if course_information placeholder is not empty" ?

Reply:
No it's the problem with djangocms template, there is not capacity to know if a placeholder is empty, the only possible way is with `{% placeholder or %}...` for default content if empty.